tests/e2e-pw/specs/merchant/merchant-orders-manual-capture.spec.ts

```diff
@@ -0,0 +1,101 @@
+/**
+ * External dependencies
+ */
+import { test, expect } from '@playwright/test';
+
+/**
+ * Internal dependencies
+ */
+import { getShopper, getMerchant } from '../../utils/helpers';
+import {
+	goToOrder,
+	goToWooPaymentsSettings,
+} from '../../utils/merchant-navigation';
+import { saveWooPaymentsSettings } from '../../utils/merchant';
+import {
+	emptyCart,
+	fillCardDetails,
+	focusPlaceOrderButton,
+	placeOrder,
+	setupProductCheckout,
+} from '../../utils/shopper';
+import { goToShop } from '../../utils/shopper-navigation';
+
+/**
+ * Local variables.
+ */
+let orderId;
+
+test.describe( 'Order > Manual Capture', () => {
+	test.beforeEach( async ( { browser } ) => {
+		// Merchant go to settings, enable capture later, and then save.
+		const { merchantPage } = await getMerchant( browser );
+		await goToWooPaymentsSettings( merchantPage );
+		await merchantPage.getByTestId( 'capture-later-checkbox' ).click();
+		await merchantPage
+			.getByRole( 'button', { name: 'Enable manual capture' } )
+			.click();
+		await saveWooPaymentsSettings( merchantPage );
+
+		// Shopper add items to cart, fill in the checkout, place an order.
+		const { shopperPage } = await getShopper( browser );
+		await emptyCart( shopperPage );
+		await goToShop( shopperPage, 1 );
+		await setupProductCheckout( shopperPage );
+		await fillCardDetails( shopperPage );
+		await focusPlaceOrderButton( shopperPage );
+		await placeOrder( shopperPage );
+
+		// Confirm that the order was placed and get the order number.
+		await shopperPage.waitForURL( /\/order-received\//, {
+			waitUntil: 'load',
+		} );
+		await expect(
+			shopperPage.getByRole( 'heading', { name: 'Order received' } )
+		).toBeVisible();
+		const orderIdField = shopperPage.locator(
+			'.woocommerce-order-overview__order.order > strong'
+		);
+		orderId = await orderIdField.innerText();
+	} );
+
+	test.afterEach( async ( { browser } ) => {
+		// Merchant go to settings, disable capture later, and then save.
+		const { merchantPage } = await getMerchant( browser );
```

Comment:
Every time we call `getMerchant` a new browser window is created, to avoid that, we can reuse the same page, doing something like:
```ts
test.describe( '...' , () => {
  let merchantPage;

  test.beforeAll( async ( { browser } => {
    merchantPage = ( await getMerchant( browser ) ).merchantPage;
}); 
```

Also notice that we could use `beforeAll` instead of `beforeEach` to call the setup part only once.

Reply:
> Every time we call `getMerchant` a new browser window is created, to avoid that, we can reuse the same page, doing something like:

I had tried to do something like this and it didn't work out the way I had hoped, so I reverted back to what I had seen in other tests. I was able to update according to your recommendation in https://github.com/Automattic/woocommerce-payments/pull/10180/commits/5b28c7bd0a1ba07110a61a667fa554c39cab209d , though, so thank you 😁 

> Also notice that we could use beforeAll instead of beforeEach to call the setup part only once.
Also updated this, as well.